Sources/RxMoya/RxMoyaProvider.swift

```diff
@@ -6,22 +6,35 @@ import Moya
 
 /// Subclass of MoyaProvider that returns Observable instances when requests are made. Much better than using completion closures.
 open class RxMoyaProvider<Target>: MoyaProvider<Target> where Target: TargetType {
+    
+    /// Propagated to Alamofire as callback queue. If nil - main queue will be used.
+    fileprivate let queue: DispatchQueue?
+    
     /// Initializes a reactive provider.
-    override public init(endpointClosure: @escaping EndpointClosure = MoyaProvider.defaultEndpointMapping,
+    public init(endpointClosure: @escaping EndpointClosure = MoyaProvider.defaultEndpointMapping,
```

Comment:
Maybe, I'm not sure 'case by some reason, queue wasn't in init. 
By the way, maybe we should create second init(return previous init, w\o queue and deprecate it? If anyone has reactive providers subclasses with overridden init - their code would not compile, sounds bad)